projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/scan-threats-status.tsx

```diff
@@ -157,8 +157,22 @@ function ThreatStatus( {
 
 	return (
 		<>
-			<div className={ baseStyles.valueSectionHeading }>
+			<div className={ clsx( baseStyles.valueSectionHeading, 'value-section__heading' ) }>
 				{ __( 'Threats', 'jetpack-my-jetpack' ) }
+				<InfoTooltip
+					tracksEventName={ 'protect_card_tooltip_open' }
+					tracksEventProps={ {
+						location: 'threats',
+						feature: 'jetpack-protect',
+						has_paid_plan: true,
+						threats: numThreats,
+					} }
+				>
+					<>
```

Comment:
Does this fragment need to be here? It's wrapped in a component, I think it can be removed 🤔

Reply:
Oh I guess not! Good catch! Thanks! 🙂